substrate/client/db/src/lib.rs

```diff
@@ -995,6 +996,10 @@ impl<Block: BlockT> sc_client_api::backend::BlockImportOperation<Block>
 		self.index_ops = index_ops;
 		Ok(())
 	}
+
+	fn create_gap(&mut self, create_gap: bool) {
```

Comment:
Nit, feel free to ignore.
```suggestion
	fn set_create_gap(&mut self, create_gap: bool) {
```